packages/vuepress-theme-odyssey/global-components/Description.vue

```diff
@@ -15,12 +15,13 @@ export default {
   props: {
     variant: {
       type: String,
-      required: false
+      required: false,
+      default: null
```

Comment:
Explicitly adding this shouldn't be necessary. I'd assume `default` is `null` by default.

Reply:
This one is weird to me as well, it is required by `eslint-config-vuepress`. It's something we should look to override in the future.